### ISSUE TITLE
Calificar el nivel del grupo de jugadores en general

### ISSUE THREAD

**Issue by LeandroRodriguez** [lgdfiuba/futbolapp-android#4]
Es una idea medio cruda, pero que los jugadores puedan puntuar tambien al "equipo" (con equipo me refiero a la gente que compone la planilla de amigos para ese partido). Esto podría ayudar en las comparativas de jugadores en diferentes partidos.
O el nivel del partido quizas se podria obtener de promediar los puntajes promedios de los que participan (o sea, el promedio de cada jugar entre todos los "equipos" en los que esta )

**Comment by imgaray** [comment]
¿La primer idea sería que cada uno califique individualmente a cada player o al equipo en general?

**Comment by LeandroRodriguez** [comment]
Al "equipo" en general. A cada player ya se lo califica, en principio, o al menos esa era la idea de nacimiento.
Ej: Juego con cierta gente los martes. Yo digo, este partido es de nivel 4. Juego con otra gente los miercoles, este partido tienen nivel 2. 
Tenes un promedio del nivel del partido segun lo que opinan los jugadores que lo componen.

**Comment by imgaray** [comment]
Entendiendo como "crew" o "grupo" al conjunto de jugadores disponibles para una clase de partidos, se podría crear alguna especie de clasificación para los equipos, como "premium", "pro", "pt" o alguna escala jerárquica similar, que sirva para poder comparar jugadores de manera más objetiva.

OBS: relacionado a issue #3

**Comment by LeandroRodriguez** [comment]
Claro. Pero esa escala la votarían los mismos integrantes de ese "grupo". Y se obtendría un promedio de eso que referencie el nivel de los partidos de ese "grupo" (y por ende de los jugadores).
